app/src/views/private/components/skip-menu.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+const { section } = defineProps<{
+	section: 'nav' | 'moduleNav' | 'main' | 'sidebar';
+}>();
+
+const { t } = useI18n();
+
+const itemsMap = {
+	nav: {
+		href: '#navigation',
+		text: t('skip_link_nav'),
+	},
+	moduleNav: {
+		href: '#module-navigation',
+		text: t('skip_link_module_nav'),
+	},
+	main: {
+		href: '#main-content',
+		text: t('skip_link_main'),
+	},
+	sidebar: {
+		href: '#sidebar',
+		text: t('skip_link_sidebar'),
+	},
+};
+
+const items = computed(() =>
+	Object.entries(itemsMap)
+		.filter(([key]) => key !== section)
+		.map(([_key, value]) => {
+			return {
+				href: value.href,
+				text: value.text,
+			};
+		}),
+);
```

Comment:
Lets simplify this by making `itemsMap` an array so you don't have to change it to one later. For example 

```suggestion
const allItems = [
	{
		id: 'nav',
		href: '#navigation',
		text: t('skip_link_nav'),
	},
	{
		id: 'moduleNav',
		href: '#module-navigation',
		text: t('skip_link_module_nav'),
	},
	{
		id: 'main',
		href: '#main-content',
		text: t('skip_link_main'),
	},
	{
		id: 'sidebar',
		href: '#sidebar',
		text: t('skip_link_sidebar'),
	},
};

const items = computed(() => allItems.filter(([key]) => key !== section));
```